src/modules/keycloak-administration/domain/keycloak-user.service.ts

```diff
@@ -273,4 +273,135 @@ export class KeycloakUserService {
 
         return { ok: true, value: userDo };
     }
+
+    public async assignRealmRolesToUser(usernameId: string, roleNames: string[]): Promise<Result<void, DomainError>> {
```

Comment:
The ticket does not say to assign the keycloak role to the user, but to assign the group.